packages/next/build/webpack/require-hook.ts

```diff
@@ -33,6 +33,30 @@ const hookPropertyMap = new Map(
       'webpack/lib/node/NodeTargetPlugin.js',
       'next/dist/compiled/webpack/NodeTargetPlugin',
     ],
+    [
+      'webpack/lib/node/NodeTemplatePlugin',
```

Comment:
Nobody noticed that these were missing because when running next.js test cases there was a `webpack` package installed. That's no longer the case (only `webpack4` and `webpack5`) so that was discovered.

Used by `worker-loader` and `@zeit/next-css`